src/gromacs/nblib/tests/testsystems.cpp

```diff
@@ -148,9 +151,22 @@ Molecule MethanolMoleculeBuilder::methanolMolecule()
 
 Topology WaterTopology::buildTopology(int numMolecules)
 {
-    // Add some molecules to the topology
+    ParticleLibrary library;
+
+    ParticleTypesInteractions interactions;
+    std::vector<std::string>  typeNames = { "Ow", "H" };
+    for (const auto& name : typeNames)
```

Comment:
Looks like this is needed repeatedly so can be a method of ParticleLibrary.

Reply:
I think better code reuse is indeed possible if combined with a test system refactoring.
Probably better addressed in a future PR